readme.txt

```diff
@@ -84,6 +84,13 @@ As of the WooCommerce 3.5 release, WooCommerce Shipping & Tax no longer provides
 
 == Changelog ==
 
+= 1.25.1 - 2020-XX-XX =
```

Comment:
Totally not a problem on this release, but one thing to note here is that it's good to keep an eye on the length of the readme.txt so the length doesn't get out of control. The changelog is displayed here: https://wordpress.org/plugins/woocommerce-services/#developers

Every couple releases, I usually end up double-checking the length and trimming off some of the oldest entries.

Reply:
Great thanks for this info, I saw it got trimmed last time so I'll keep it in mind for the future releases.